### PR TITLE
added page to ask username from user

### LINKED ISSUE #7: Add a page to ask for Username
**Note:** This issue is reserved for SLoP Contributors.

There is no login or sign up. The username is how Sinix will identify the user.

### PR DESCRIPTION
**Pull request by GajjarMiten**
## Description 

added `OnBordingPage()` for asking username from the user and updating `username` in `store.dart`.

## Related Issue

Fixes #7

### CODE REVIEW

**Review by janvig — COMMENTED** [review]
Works fine :+1:

**Review by janvig — CHANGES_REQUESTED** [review]
Please fix the spelling of "onboarding" wherever spelled incorrectly.

**Comment by GajjarMiten** [comment]
Hey @janvig!
could you please point out where should I fix a typo? Because I couldn't find any.

File: lib/main.dart [hunk 1]
```diff
@@ -28,7 +29,7 @@ class App extends StatelessWidget {
       home: MaterialApp(
         theme: ThemeData(fontFamily: 'Roboto'),
         home: Scaffold(
-          body: DiscoverDevices(),
+          body: OnBordingPage(),
```
Comment (sanket143) [inline lib/main.dart]: It should be `OnBoardingPage`

File: lib/utils/store.dart [hunk 2]
```diff
@@ -41,9 +41,9 @@ class Store extends GetxController {
 
     if (response.statusCode == 200) {
       isConnectionError = false;
-      this.user.connect(username);
+      this.user.connect(user.username);
       this.channel =
-          IOWebSocketChannel.connect("ws://$ipAddr:41431/ws/$username");
+          IOWebSocketChannel.connect("ws://$ipAddr:41431/ws/$user.username");
```
Comment (sanket143) [inline lib/utils/store.dart]: Since we need the value of `username`, it'll be `${user.username}` instead of `$user.username`.

**Review by sanket143 — CHANGES_REQUESTED** [review]

File: lib/utils/store.dart (hunk 2 above)
Comment (GajjarMiten) [reply lib/utils/store.dart]: Hey @sanket143!
I have made the changes!
could you please review it?

**Comment by GajjarMiten** [comment]
Hey @sanket143 @janvig
could you please look into this pr?

**Comment by sanket143** [comment]
give @GajjarMiten 75 points

**Pull request merged**